frontend/package.json

```diff
@@ -108,7 +108,7 @@
     "npm-run-all2": "^7.0.0",
     "nuxt": "^3.14.1592",
     "rimraf": "^6.0.1",
-    "storybook": "^8.3.6",
+    "storybook": "8.3.6",
```

Comment:
The CI was failing because it was using the latest Storybook version

Reply:
What a weird error to have!